modules/saml/lib/IdP/SAML2.php

```diff
@@ -376,6 +401,12 @@ public static function receiveAuthnRequest(SimpleSAML_IdP $idp)
             'idpInit'     => $idpInit,
         ));
 
+        if ($binding instanceof SAML2_SOAP) {
+            $supportedBindings = array(
+                \SAML2\Constants::BINDING_PAOS,
+            );
```

Comment:
I think this could cause a problem if we're expected to respond on a different binding. We can simply append this to `supportedBindings` if `saml20.ecp` is enabled, up where we set the others.